code/modules/mob/living/carbon/human/species/outsider/skeleton.dm

```diff
@@ -92,7 +92,7 @@
 
 	remains_type = /obj/effect/decal/cleanable/ash
 
-	flesh_color = "#AAAAAA"
+	flesh_color = "#551A8B"
```

Comment:
Flesh color is only used for the text in the species name at the examine and for creating gibs.